### PR TITLE
CB-230: Migrate ws, frontend and db completely off the ORM

### PR DESCRIPTION
**Pull request by ferbncode**
Made changes to `frontend`, `db` and `ws` code to use migrated code. Also added `drop_tables.sql` and `drop_types.sql` to help `setUp()` and `tearDown()` in tests. Only functions remaining in `dump_manager.py` use ORM related code.

### CODE REVIEW

File: critiquebrainz/data/testing.py [hunk 1]
```diff
@@ -14,9 +14,13 @@ def create_app(self):
...
 
     def setUp(self):
-        db.create_all()
+        self.reset_db()
         # TODO(roman): Add stuff form fixtures.
 
     def tearDown(self):
-        db.session.remove()
-        db.drop_all()
+        pass
```
Comment (gentlecat) [inline critiquebrainz/data/testing.py]: Seems like `tearDown` can be just removed there if it's not needed.

File: critiquebrainz/frontend/testing.py [hunk 2]
```diff
@@ -14,12 +14,16 @@ def create_app(self):
...
 
     def setUp(self):
-        db.create_all()
+        self.reset_db()
         # TODO(roman): Add stuff form fixtures.
 
     def tearDown(self):
-        db.session.remove()
-        db.drop_all()
+        pass
```
Comment (gentlecat) [inline critiquebrainz/frontend/testing.py]: Same here. If `tearDown` is not needed it can just be removed.

File: critiquebrainz/ws/testing.py [hunk 3]
```diff
@@ -18,12 +18,16 @@ def create_app(self):
...
 
     def setUp(self):
-        db.create_all()
+        self.reset_db()
         # TODO(roman): Add stuff form fixtures.
 
     def tearDown(self):
-        db.session.remove()
-        db.drop_all()
+        pass
```
Comment (gentlecat) [inline critiquebrainz/ws/testing.py]: And here.

File: manage.py [hunk 4]
```diff
@@ -102,7 +102,7 @@ def init_db(skip_create_db=False):
         create_extension(db_uri)
 
     click.echo("Creating tables... ", nl=False)
-    data_utils.create_tables()
+    data_utils.create_all()
```
Comment (gentlecat) [inline manage.py]: Have you tested this on a new instance? Does it work fine?

Comment (ferbncode) [inline manage.py]: Yes, I have tested this on a new instance and it works fine 🙂

**Pull request merged**